docs/docs/frontend/project.md

```
@@ -0,0 +1,59 @@
+---
+sidebar_position: 3
+---
+
+# 📂 Projeto
+
+### Como Executar o Projeto
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)